docs/src/man/joins1.md

```diff
@@ -0,0 +1,609 @@
+# Joining DataFrames
+
+"I have two different data sets in Julia but I am not sure how to use them. What 
```

Comment:
maybe be more precise in "to use them" and vaguely describe what you want to do with them?